src/models/migrations/20200827192413-create-user.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Users', {
+    userId: {
+      type: Sequelize.INTEGER,
+      allowNull: false,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    name: {
```

Comment:
I'm possibly overthinking this for a sprint 0 ticket but we might want `first` `last` as separate cols. The UI might want to do welcome and user messages with just the first name.  Trying to split out a combined field on the frontend is usually error prone.

Reply:
Yes, `name `was meant more of a placeholder, but if we know how the data is going to be structured, will change it. I'll investigate.